### PR TITLE
Removing AppCompatActivity in the CallQuery method

### PR DESCRIPTION
**Pull request by adfleshner**
Removed the AppComaptActivity Dependency for the callQuery method.
Using Handler and post instead.

### CODE REVIEW

**Comment by adfleshner** [comment]
By doing is this way you can easily use this on a ViewModel without needing access to the Activity.

**Comment by ShwetaChauhan18** [comment]
@adfleshner : Thanks for your support. I will review and let you know.

File: SSffmpegVideoOperation/src/main/java/com/simform/videooperations/CallBackOfQuery.kt [hunk 1]
```diff
@@ -34,30 +36,39 @@ public class CallBackOfQuery {
...
             }
             else -> {
-                ffmpegCallBack.failed()
-                Config.printLastCommandOutput(Log.INFO)
-            }
+                processHandler.post {
+                    ffmpegCallBack.failed()
+                    Config.printLastCommandOutput(Log.INFO)
+                }
+                }
```
Comment (AshvinVavaliya) [inline SSffmpegVideoOperation/src/main/java/com/simform/videooperations/CallBackOfQuery.kt:71]: Please do the code in format.

Comment (ShwetaChauhan18) [reply SSffmpegVideoOperation/src/main/java/com/simform/videooperations/CallBackOfQuery.kt:71]: @adfleshner : Also, please update library version

Comment (ShwetaChauhan18) [reply SSffmpegVideoOperation/src/main/java/com/simform/videooperations/CallBackOfQuery.kt:71]: @adfleshner: Also, I have merged 1 PR, so rebase your code with latest one

**Comment by adfleshner** [comment]
So I rebased up until now if there is anything else let me know if there is anything else you need.

**Review by mobile-simformsolutions — APPROVED** [review]

**Pull request merged**